cadence/contracts/GametvNFT.cdc

```diff
@@ -20,12 +22,12 @@ pub contract GametvNFT: NonFungibleToken {
     // The total number of GametvNFTs that have been minted
     pub var totalSupply: UInt64
     
-    // NFT type schema
+    // NFT type
+    // It is used to keep a check on the current NFTs minted and the total number of NFTs that can be minted
+    // for a given type
     pub struct NftType {
         pub let typeId : String
 
-        pub var metaData : {String : AnyStruct}
-
         pub var currentCount : UInt64
 
         pub let maxCount : UInt64
```

Comment:
Can we make the syntax consistent everywhere and have no space before `:` and one space after? For example, this would become `pub let maxCount: UInt64`

Reply:
I think I saw this somewhere thats why did it. Never mind will make it consistent